README.md

```diff
@@ -80,6 +82,22 @@ Here's an example of a custom event handler you can extend to
 process the returned transcription results as needed. This
 handler will simply print the text out to your interpreter.
 """
+
+
+SAMPLE_RATE = 16000
+BYTES_PER_SAMPLE = 2
+CHANNEL_NUMS = 1
+
+# An example file can be found at tests/integration/assets/test.wav
+# NOTE: For pre-recorded files longer than 5 minutes, the sent audio
+# chunks should be rate limited to match the realtime bitrate of the
+# audio stream to avoid signing issues.
+AUDIO_PATH = "test.wav"
```

Comment:
Was there a reason we changed the pathing on this? I'm not sure if the file is reachable this way from the top level directory.